src/current/cockroachcloud/ccloud-get-started.md

```diff
@@ -45,11 +45,18 @@ If you are a member of more than one [CockroachDB Cloud organization](https://ww
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-ccloud auth login --org <organization label>
+ccloud auth login --org {organization label}
 ~~~
 
 The organization label is found on the **Settings** page of the CockroachDB Cloud Console.
 
+If your organization has a custom URL, use the `--vanity-name` flag to log in:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud auth login --vanity-name {custom organization URL}
```

Comment:
```suggestion
ccloud auth login --vanity-name {custom-organization-name}
```

Replace `{custom-organization-name}` with the final portion of the custom sign-in URL, after `/login/`. Do not pass the full custom sign-in URL.